### PR TITLE
[DOCS]: CONTRIBUTING.md and CODE_OF_CONDUCT.md files added

### LINKED ISSUE #87: [Docs]: Setup a contributing md file for new contributors
## Description: 

PomoSuperFocus is on initial phase of development and we need to setup a contributing guide for newcomers! 💙

### PR DESCRIPTION
**Pull request by Lagaress**
Fixes #87

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# Contributor Code of Conduct
+PomoSuperFocus is a free and open source project aimed at helping people stay focused and productive. While the project currently operates with some technical limitations due to using free-tier hosting services (Render's Free Hobby plan for backend and MongoDB's 500MB free tier for database), we remain committed to providing value to our community through features like Guest Login and Beta access.
```
Comment (samyakshah3008) [inline CODE_OF_CONDUCT.md:2]: We have shifted to Vercel Free plan so can you please change "Render" to "Vercel" ?

File: CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+Examples of unacceptable behavior include:
+
+* The use of sexualized language or imagery, and sexual attention or advances of any kind
+* Trolling, insulting or derogatory comments, and personal or political attacks
+* Public or private harassment
+* Publishing others' private information, such as a physical or email address, without their explicit permission
+* Other conduct which could reasonably be considered inappropriate in a professional setting
+* Not feeding stray cats and animals 😿💔
+
```
Comment (samyakshah3008) [inline CODE_OF_CONDUCT.md:32]: I liked you added this, but not everyone is animal lover 😄 so please remove this point as we cannot add in unacceptable behavior however it's encouraged so...

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+6. [Pull Request Process](#pull-request-process)
+7. [Community](#community)
+
+## Code of Conduct
+
+Please note that this project is released with a [Contributor Code of Conduct](./CODE_OF_CONDUCT.md). By participating in this project, you agree to abide by its terms.
+
+## How to Contribute
+
+1. **Join the Community**: Join our [Discord server](https://discord.gg/HmprpsBb) to connect with other contributors and discuss ideas.
```
Comment (samyakshah3008) [inline CONTRIBUTING.md:23]: Latest discord server link is `https://discord.gg/CNncC5XD89` can you please update?

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+## Pull Request Process
+
+1. **Create a branch**: Use a descriptive name for your branch. We follow the convention of `type/description` (e.g., `fix/bug-123` or `feat/new-feature`). You can see the existing branches [here](https://github.com/samyakshah3008/pomo-super-focus-webapp/branches) to get an idea.
+2. **Commit your changes**: Write clear and concise commit messages.
+3. **Push to GitHub**: Push your branch to your forked repository.
+4. **Open a pull request**: Submit your pull request to the `master` branch of the original repository.
+
+## Community
+
+- **Discord**: Join our [Discord server](https://discord.gg/HmprpsBb) for real-time discussions.
```
Comment (samyakshah3008) [inline CONTRIBUTING.md:54]: For discord link please add the latest link -  `https://discord.gg/CNncC5XD89`

File: README.md [hunk 5]
```diff
@@ -8,16 +8,14 @@ In our day-to-day lives, people use a variety of tools to stay organized and pro
 
 ### Challenges:
 
-PomoSuperFocus is free to use and open source forever, however - currently I have hosted my backend on Vercel with Free Hobby plan until I make a purchase of a paid plan. Due to this, this Free plan cannot handle heavy traffic and has certain constraints. Along with this, I am using MongoDB for Database which is also on a free plan with a limit of 500mb only. To handle this situation and make a workaround, I came up with a solution - Guest Login and Beta access upon request.
+PomoSuperFocus is free to use and open source forever, however - currently I have hosted my backend on Render with Free Hobby plan until I make a purchase of a paid plan. Due to this, this Free plan cannot handle heavy traffic and has certain constraints. Along with this, I am using MongoDB as Database which is also on a free plan with a limit of 500mb only. To handle this situation and make a workaround, I came up with a solution - Guest Login and Beta access upon request.
```
Comment (samyakshah3008) [inline README.md:11]: Please sync your master branch @Lagaress with my master branch as I already made necessary changes for README file, your changes are overriding them here.

**Review by samyakshah3008 — COMMENTED** [review]
Great work @Lagaress ! Thanks for the work, loved the descriptive content. Have added a few comments - please check them! and then we are good to go! ⚡

File: CODE_OF_CONDUCT.md (hunk 1 above)
Comment (Lagaress) [reply CODE_OF_CONDUCT.md]: Done!

File: CODE_OF_CONDUCT.md (hunk 2 above)
Comment (Lagaress) [reply CODE_OF_CONDUCT.md]: Done!

File: CONTRIBUTING.md (hunk 3 above)
Comment (Lagaress) [reply CONTRIBUTING.md]: Done!

File: CONTRIBUTING.md (hunk 4 above)
Comment (Lagaress) [reply CONTRIBUTING.md]: Done!

File: README.md (hunk 5 above)
Comment (Lagaress) [reply README.md]: Done!

**Review by samyakshah3008 — APPROVED** [review]
Loved it @Lagaress ! Thanks for your contribution!

**Pull request merged**